app/queues/notifications/record-expiration-monitor.server.ts

```diff
@@ -0,0 +1,52 @@
+import { Queue } from 'bullmq';
+import { redis } from '~/lib/redis.server';
+import logger from '~/lib/logger.server';
+import type { DomainStatusData } from './record-cert-expiration-data.interface';
+import { prisma } from '~/db.server';
+import {
+  startRecordExpirationMonitorWorker,
+  recordExpirationQueueName,
+} from './record-expiration-monitor.worker';
+
+const { NOTIFICATIONS_REPEAT_PATTERN } = process.env;
+
+// fetch records with by status
+const getRecordsByExpiration = async () => {
+  let monthFromToday = new Date();
+  monthFromToday.setMonth(monthFromToday.getMonth() + 1);
+  return await prisma.record.findMany({
```

Comment:
Same issue with `async` and `await`.  Also, this code is very similar to the above, with only the query being different.  We can consolidate this into a single worker/queue that does a different query.